carbon_bombs/io/neo4j.py

```diff
@@ -435,3 +436,20 @@ def delete_all(tx):
     LOGGER.debug("NEO4J purge done")
 
     driver.close()
+
+
+def create_local_database():
+    """Create local database with companies, banks, countries and carbon bombs"""
+    LOGGER.debug("Update cleaned csv for neo4j and save them")
+    carbon_bombs, companies, banks, countries = update_csv_nodes_neo4j()
+
+    # Define dict to iterate over three types node creation
+    dict_nodes = {
+        "carbon_bomb": carbon_bombs.to_dict(orient='records'),
+        "company": companies.to_dict(orient='records'),
+        "bank": banks.to_dict(orient='records'),
+        "country": countries.to_dict(orient='records'),
+    }
+
+    with open('database.json', 'w') as convert_file: 
```

Comment:
Put the file name in `conf.py`
Also maybe use : the following path `{DATA_NEO4J_PATH}/database.json`